modules/adding-cluster-notification-contacts.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_admin/osd_logging/osd-accessing-the-service-logs.adoc
+// * rosa_cluster_admin/rosa_logging/rosa-accessing-the-service-logs.adoc
+
+:_content-type: PROCEDURE
+[id="adding-cluster-notification-contacts_{context}"]
+= Adding cluster notification contacts
+
+You can add notification contacts for your 
+ifdef::openshift-dedicated[]
+{product-title} 
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA) 
+endif::openshift-rosa[]
+cluster. When an event occurs that triggers a cluster notification email, subscribed users are notified.
+
+.Procedure
+
+. Navigate to {cluster-manager-url} and select your cluster.
+
+. On the *Support* tab, under the *Notification contacts* heading, click *Add notification contact*.
+
+. Enter the Red Hat username or email of the contact you want to add.
```

Comment:
suggesting clarify that the username or email should belong to the same organizations

Reply:
@tzhou5 thank you for this suggestion. I have subsequently added the following note after this step:

> **Note**: The username or email address must relate to a user account in the Red Hat organization where the cluster is deployed.